web/webpack.config.js

```diff
@@ -89,6 +89,18 @@ const copyWebpackCommands = [
     from: `${STATIC_ROOT}/cast/`,
     to: `${output.PATH}/public/cast/`,
   },
+  {
+    from: `${STATIC_ROOT}/img/moon.png`,
+    to: `${output.PATH}/public/img/moon.png`,
+  },
+  {
+    from: `${STATIC_ROOT}/img/my_channels.jpg`,
+    to: `${output.PATH}/public/img/my_channels.jpg`,
+  },
+  {
+    from: `${STATIC_ROOT}/img/my_pledges.jpg`,
+    to: `${output.PATH}/public/img/my_pledges.jpg`,
+  },
```

Comment:
Please remove the actual files from git (move to CDN) and squash before it goes into `master` -- otherwise, it stays in the repo and makes the cloning process longer than necessary if accumulated.

Understand it is for quick development, so should be fine as long as it doesn't reach `master`.

Reply:
This isn't supposed to get merged into master. All the images are still placeholders.